### PR TITLE
Issue21 post format

### PR DESCRIPTION
**Pull request by CunliangGeng**
Try to solve issue #21.

In addition, added the validation for unwanted RDF types in metadata to avoid the mixing of different layers of metadata, e.g. mixing fdp and catalog metadata in one input.

### CODE REVIEW

File: fdp/fdp.py [hunk 1]
```diff
@@ -157,21 +186,14 @@ def get(self):
         '''
         Get the list of catalog URIs
         '''
-        return httpResponceNav(app.graph, 'Catalog')
+        return httpResponceNav('Catalog')
```
Comment (c-martinez) [inline fdp/fdp.py]: httpResponceNav -> httpResponseNav

File: fdp/fdp.py [hunk 2]
```diff
@@ -57,11 +66,17 @@ def httpResponse(graph, uri):
 
     return resp
 
-def httpResponceNav(graph, layer):
-    """HTTP response: metadata navigations"""
+def httpResponceNav(layer):
```
Comment (c-martinez) [inline fdp/fdp.py]: httpResponceNav -> httpResponseNav

File: tests/data/fdp_invalid_missingRequired.ttl [hunk 3]
```diff
@@ -1,11 +1,11 @@
 @prefix ex: <http://example.com/ns#> .
 @prefix fdp: <http://rdf.biosemantics.org/ontologies/fdp-o#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dct: <http://purl.org/dc/terms/> .
```
Comment (c-martinez) [inline tests/data/fdp_invalid_missingRequired.ttl]: I am not sure what is more common, if using the prefix `dct` or `dcterms`. In practice it doesn't make any difference, but I think I have seen `dcterms` more frequently. What do you think?

Comment (CunliangGeng) [reply tests/data/fdp_invalid_missingRequired.ttl]: You're right. We should use `dcterms` to be consistent with the extended [default namespaces](https://rdflib.readthedocs.io/en/stable/_modules/rdflib/namespace.html) in the library `rdflib v5.0`. The rdflib v5.0 was released a month ago. We're currently still using `rdflib v4.2.2` which does not have the namespace 'dcterms'.

File: fdp/fdp.py (hunk 2 above)
Comment (CunliangGeng) [reply fdp/fdp.py]: Thanks, fixed!

File: tests/data/fdp_invalid_missingRequired.ttl (hunk 3 above)
Comment (c-martinez) [reply tests/data/fdp_invalid_missingRequired.ttl]: Ah good find. Maybe we should upgrade to 5.0.

**Review by c-martinez — APPROVED** [review]

**Pull request merged**